db/migrate/20160701161129_remove_non_alphanumeric_chars_from_developer_username.rb

```diff
@@ -0,0 +1,13 @@
+class RemoveNonAlphanumericCharsFromDeveloperUsername < ActiveRecord::Migration
+  def up
+  	Developer.find_each do |developer|
+	  if developer.username != developer.username.gsub(/[^0-9a-z ]/i, '')
+	  	developer.username = developer.username.gsub(/[^0-9a-z ]/i, '')
+	    developer.save!
+	  end
+  	end
+  end
+
+  def down
```

Comment:
This method should do something. Since we are permanently changing data, perhaps raising `ActiveRecord::IrreversibleMigration` would work?